www/docs/mp/index.php

```diff
@@ -219,37 +219,28 @@
     $title .= ' MSP, '.$MEMBER->constituency();
 }
 
-$positions = array();
-
-// Position if this is a member of the Commons
-if ($MEMBER->house(HOUSE_TYPE_COMMONS)) {
-    $position = $MEMBER->current_member(HOUSE_TYPE_COMMONS) ? 'MP' : 'Former MP';
-    if ($MEMBER->constituency()) $position .= ', ' . $MEMBER->constituency();
-    $positions[] = $position;
+$known_for = '';
+$current_offices_ignoring_committees = $MEMBER->offices('current', TRUE);
+if (count($current_offices_ignoring_committees) > 0) {
+    $known_for = $current_offices_ignoring_committees[0];
 }
 
-// Position if this is a member of NIA
-if ($MEMBER->house(HOUSE_TYPE_NI)) {
-    $position = $MEMBER->current_member(HOUSE_TYPE_NI) ? 'MLA' : 'Former MLA';
-    if ($MEMBER->constituency()) $position .= ', ' . $MEMBER->constituency();
-    $positions[] = $position;
-}
-
-// Position if this is a member of Scottish Parliament
-if ($MEMBER->house(HOUSE_TYPE_SCOTLAND)) {
-    $position = $MEMBER->current_member(HOUSE_TYPE_SCOTLAND) ? 'MSP' : 'Former MSP';
-    $position .= ', ' . $MEMBER->constituency();
-    $positions[] = $position;
-}
-
-$position = implode('; ', $positions);
-
-$current_offices = $MEMBER->offices('current', TRUE);
-$former_offices = $MEMBER->offices('previous', TRUE);
-
-// If this person has named non-committee offices, they override the default
-if (count($current_offices) > 0) {
-    $position = $current_offices[0];
+$role = '';
+foreach( array(
+    'Peer' => HOUSE_TYPE_LORDS,
```

Comment:
This will produce output such as "Former Bishop Peer" which makes little sense.

Reply:
Aha, I see elsewhere in this file, we specially handle Bishops so they don’t get treated as a party – https://github.com/mysociety/theyworkforyou/blob/98134134cb0730e66aea4b06c9a6654572b7bbf3/www/docs/mp/index.php#L814 – I’ll try that here too.